File: ProjetoModelo/DAO/Purchase.cs
namespace DAO
{
    public class Purchase
    {
        public int ID;
        public DateTime purchase_date;
        public string? confirmation_number;
        public string? number_nf;
        public int Payment;
        public int PurchaseStatus;
        public double purchase_value;

        public Client? client;

        public Product? product;

        public Store? store;
    }
}

File: projeto-front/src/app/purchase.ts
import { Product } from './products';
import { Store } from './store';
export interface Purchase
{
    id: number;
    payment: number;
    confirmationNumber: number;
    product: Product;
    store: Store;
    client: string;
    purchaseDate: Date;
    purchaseValue: number;
}